### PR TITLE
chore: add "Going further" section for adding Prisma Migrate to an existing project

### LINKED ISSUE #3098: Add a link about deploying at the end of Migrate Guide
As Austin pointed out, the current [Migrate Guide](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate) leaves you hanging about the next big step of deploying to production.

See [this Slack post](https://prisma-company.slack.com/archives/C5Z9TH6N9/p1651008110783959):
_"I think it would make sense to tack on another section to [this Prisma Migrate guide](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate). We take the user through the flow of editing their schema and creating migrations, but we don’t link or reference prisma migrate deploy anywhere.
A simple “Ready to go to production?” with a link to [this Migrate deploy guide](https://www.prisma.io/docs/guides/deployment/deploy-database-changes-with-prisma-migrate) could be enough. Thoughts?"_

### PR DESCRIPTION
**Pull request by ruheni**
## Describe this PR

Add a going further section to Adding Prisma Migrate to an existing project

Closes #3098

### CODE REVIEW

File: content/300-guides/025-migrate/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx [hunk 1]
```diff
@@ -120,11 +120,7 @@ Commit the following to source control:
...
+## Going further
 
-In certain cases, you can run into issues with the migration history and that prevent you from applying further database migrations with `prisma migrate resolve`. This can happen if:
-
-- A migration failed, either because of an error or because it was interrupted while running (for example, unexpected shutdown)
-- The database needs to be baselined to skip certain migrations that are unnecessary on this database schema - this can be the case when a change to the database schema was done manually, such as a hotfix.
-
-In order to fix these situations have a look at [Production Troubleshooting](/guides/migrate/production-troubleshooting#fixing-failed-migrations-with-migrate-diff-and-db-execute)
+- Refer to the [Deploying database changes with Prisma Migrate](/guides/migrate/deployment/deploy-database-changes-with-prisma-migrate) guide for more on deploying migrations to production.
+- Refer to the [Production Troubleshooting](/guides/migrate/production-troubleshooting#fixing-failed-migrations-with-migrate-diff-and-db-execute) guide to learn how to debug migration issues in production.
```
Comment (jharrell) [inline content/300-guides/025-migrate/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx]: Maybe add more here? Something like "Including failed migrations and issues with `prisma migrate resolve`" just so if someone searches for that they find a solution.

Comment (ruheni) [inline content/300-guides/025-migrate/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx]:
```suggestion
- Refer to the [Production Troubleshooting](/guides/migrate/production-troubleshooting#fixing-failed-migrations-with-migrate-diff-and-db-execute) guide to learn how to debug and resolve failed migrations in production using `prisma migrate diff` and `prisma db execute`.
```

Comment (ruheni) [reply content/300-guides/025-migrate/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx]: I've updated it. Does it read better?

Comment (jharrell) [reply content/300-guides/025-migrate/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx]: Great 👍

**Review by jharrell — APPROVED** [review]

File: content/300-guides/025-migrate/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx [hunk 2]
```diff
@@ -120,11 +120,7 @@ Commit the following to source control:
...
+## Going further
 
-In certain cases, you can run into issues with the migration history and that prevent you from applying further database migrations with `prisma migrate resolve`. This can happen if:
-
-- A migration failed, either because of an error or because it was interrupted while running (for example, unexpected shutdown)
-- The database needs to be baselined to skip certain migrations that are unnecessary on this database schema - this can be the case when a change to the database schema was done manually, such as a hotfix.
-
-In order to fix these situations have a look at [Production Troubleshooting](/guides/migrate/production-troubleshooting#fixing-failed-migrations-with-migrate-diff-and-db-execute)
+- Refer to the [Deploying database changes with Prisma Migrate](/guides/migrate/deployment/deploy-database-changes-with-prisma-migrate) guide for more on deploying migrations to production.
+- Refer to the [Production Troubleshooting](/guides/migrate/production-troubleshooting#fixing-failed-migrations-with-migrate-diff-and-db-execute) guide to learn how to debug and resolve failed migrations in production using `prisma migrate diff` and `prisma db execute`.
```
Comment (Jolg42) [inline content/300-guides/025-migrate/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx]:
```suggestion
- Refer to the [Production Troubleshooting](/guides/migrate/production-troubleshooting#fixing-failed-migrations-with-migrate-diff-and-db-execute) guide to learn how to debug and resolve failed migrations in production using `prisma migrate diff`, `prisma db execute` and/ or `prisma migrate resolve`.
```

**Pull request merged**